content/docs/setup/kubernetes/platform-setup/index.md

```diff
@@ -7,21 +7,6 @@ keywords: [kubernetes]
 
 Follow these instructions to setup the Kubernetes cluster for Istio.
 
-## Prerequisites
```

Comment:
can you revert this part of the PR?  I prefer the prerequisites for the kubernetes versions in platform-setup as it reads more smoothly.

Reply:
makes sense.  I'm not sure what state this PR is in given all the construction around platform setup.  I added a needs-rebase, so any remaining work items you wanted to tackle can be consolidated on top of master.  Please assign to me and I will process quickly.

Reply:
The prerequisites have been removed from the master branch. And I think it looks fine. I plan to close my PR